msal_extensions/windows.py

```diff
@@ -39,6 +39,15 @@ def raw(self):
         _MEMCPY(blob_buffer, pb_data, cb_data)
         return blob_buffer.raw
 
+_err_description = {
+    # Keys came from real world observation, values came from winerror.h (http://errors)
```

Comment:
Then I think we should let the reader know this link is Microsoft internal, as the MSAL library and its code is public.

```suggestion
    # Keys came from real world observation, values came from winerror.h (http://errors (Microsoft internal))
```